src/components/ecl-button/src/styles/ecl-button-eu.scss

```diff
@@ -1,35 +1,23 @@
-@use '@ecl/theme-eu';
-@use '@ecl/vanilla-component-icon/icon-eu';
-@use '@ecl/vanilla-component-link/link-eu';
-@use '@ecl/vanilla-component-button/button-eu';
+@use '@ecl/theme-eu/theme' as theme;
+@use '@ecl/theme-eu/variables' as var;
+@use '@ecl/grid/grid' with (
```

Comment:
ah, but why do we need this here..? i mean, it's an inner dependency of the button vanilla component, we shouldn't need to add it here, can you try without it? Also removing the dependency in the package.json.

Reply:
removed